news_items/043_units_element/index.html

```diff
@@ -0,0 +1,68 @@
+<style>
+  img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  pre {
+    width: 90%;
+    margin-left: auto;
+    margin-right: auto;
+    overflow: hidden;
+    margin-bottom: 1rem;
+    font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+    background: #f0f0f0;
+    padding: 1em;
+    padding-bottom: 0em;
+  }
+
+</style>
+
+<p>
+  PrairieLearn now has a new <code class="user-output">pl-units-input</code> element, due to
+  <a href="https://github.com/eliotwrobson">Eliot Robson</a> (UIUC).
+</p>
+
+<p>
+  This element can be used to create problems where students must input a quantity with an accompanying unit.
+  By default, input of the same dimensionality as the reference solution is automatically converted and compared.
+</p>
+
+<img class="border rounded" src="pl-units-input.png" width="800px">
+
+<p>Below are submission panels for correct answers. For the second one, the comparison was performed independent of
+  units and within the given tolerance (1cm).</p>
+
+<img class="border rounded" src="pl-units-input-correct.png" width="400px">
+<img class="border rounded" src="pl-units-input-correct-2.png" width="400px">
+
+<p>Note in the following example that the provided submission had incorrect dimensionality, and no comparison could be
+  performed.</p>
+
+
+<img class="border rounded" src="pl-units-input-incorrect.png" width="600px">
+
+
+<p>
+  Additional comparison methods are available without this automated conversion, along with other customization
+  options. The code that generates the above question is:
+</p>
+
+<div class="code-block rounded">
```

Comment:
Missing closing tag for this?

Reply:
Removed in https://github.com/PrairieLearn/PrairieLearn/pull/7302/commits/1e5ae79d638a7dbec353a9c054e9d533f1bd3fa0